packages/core-toggle/core-toggle.js

```diff
@@ -27,12 +27,13 @@ export default function toggle (buttons, open) {
 addEvent(UUID, 'keydown', (event) => {
   if (event.keyCode === KEYS.ESC) {
     for (let el = event.target; el; el = el.parentElement) {
-      const prev = el.previousElementSibling
-      const inPopup = prev && prev.hasAttribute(UUID) && (el = prev)
-      if (inPopup || el.hasAttribute(UUID)) {
-        const open = el.getAttribute(OPEN) === 'true'
-        const pops = el.getAttribute(POPS) === 'true'
-        if (open && pops) return setOpen(el, false, el.focus())
+      const uuid = el.hasAttribute(UUID) || (el = el.previousElementSibling || el).hasAttribute(UUID)
```

Comment:
`(el = el.previousElementSibling || el).hasAttribute(UUID)`:

når `el = el.previousElementSibling` er undefined, kommer ikke denne til å kaste exception `cannot read hasAttribute of undefined`?

Reply:
som er null (satt av el = el.previousElementSibling)

Reply:
nei vent. Jeg er treig.. du har rett.

Reply:
poenget blir vel heller at det er tungt å lese ;)